.github/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Release to NPM
+on:
+  push:
+    branches:
+      - "main"
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Use Node 12
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12
+        registry-url: 'https://registry.npmjs.org'
+        scope: '@eclipse-che'
+
+    - name: Install dependencies
+      run: yarn
+
+    - name: Build extension
+      run: yarn build
+
+    - name: Run Test
+      run: yarn test
+
+    - name: Publish to npm
+      run: |
+        git config --global user.email "che-bot@eclipse.org"
+        git config --global user.name "CHE Bot"
+        
+        yarn --new-version version "0.0.1-$(date +%s)"
```

Comment:
previously we had `npm version -no-git-tag-version "0.0.1-$(date +%s)" && npm publish --access public` 
I'm not sure if there is any difference, should be move back or do the same as devworkspace client does.